overlay_new.py

```diff
@@ -76,6 +86,12 @@ def data_field_coord(x, y):
                 self.get_data_func("reed_distance", 2, 0.001),
                 data_field_coord(3, 1),
             ),
+            VoltageField(
+                # should change colour at 3.5V
```

Comment:
This comment is incorrect - although tbh it's probably not relevant here anyway.